### ISSUE TITLE
[Feature Request] Assets Bar

### ISSUE THREAD

**Issue by MattisBrumm** [amidaware/tacticalrmm#1635]
**Is your feature request related to a problem? Please describe.**
If I select a worksation and click on Assets, there is a bar with Operating System, CPU, .... This bar is grey.
On every other Tab (in Checks, Tasks, Patches, ...) for example Notes there is another Bar with Refresh and Add Note, which is in black or white. (depends on white and dark mode)

So it would be nice if the bar with Operating System, CPU, .... would be black/white too and not grey.
Also this bar is not fixed to the top like if you scroll through the Checks and the "Run Checks Now" still is at the top.

Would be great to change these maybe?

**Comment by wh1te909** [comment]
yes you want it in a table basically with columns and rows, just like the other tabs.

**Comment by rubybrars** [comment]
Actually it is white and blue based on which one is open
